docs/eve-images.md

```diff
@@ -131,3 +131,9 @@ When done, you have a live image file to be used, normally in `dist/<arch>/<file
 To launch eden using the provided image:
 
 1. `eden start --image-file=path/to/your/live-image`
+
+### Overwrite config of EVE
+
+You can add files into config partition of EVE (along with the files that are generated by EdenEden) by copying them into `conf` directory.
```

Comment:
Let's rename it to eve-conf or something... Misleading with eden config which is in /.eden

Reply:
Yeah -- big +1 to renaming

Reply:
done